src/main/java/org/liquidengine/legui/layout/Layout.java

```diff
@@ -0,0 +1,33 @@
+package org.liquidengine.legui.layout;
+
+import org.liquidengine.legui.component.Component;
+
+/**
+ * Layout interface.
+ * @author Aliaksandr_Shcherbin.
+ */
+public interface Layout {
+
+    /**
+     * Used to add component to layout.
+     *
+     * @param component component to add.
+     * @param constraint layout constraint.
+     */
+    void addComponent(Component component, LayoutConstraint constraint);
```

Comment:
What happens if you use a LayoutConstraint not supported by this Layout? maybe there should be an exception InvalidLayoutConstraint or at least say that the implementation must specify a default behaviour

Reply:
I'll add description to interface.